apps/terminal.py

```diff
@@ -75,6 +75,10 @@ def ssh_copy_id_servers(m):
 
 
 keymap = {
+    "shell Whereami": "pwd ",
+    "shell home": "~/",
+    "shell changedir": "cd",
+    "shell copy": "cp",
```

Comment:
this is already implemented in this file on line 124, unless you are intending different semantics

Reply:
you are correct. sorry, i'll remove it from the PR